### PR TITLE
JBPM-5071 - remove of jbpm runtime from workbench

### PR DESCRIPTION
**Pull request by mswiderski**
- Migrate executor service from Kie to Uberfire
- clean up of asset mgmt - leaving only create repository with branches

### CODE REVIEW

**Comment by mrietveld** [comment]
+1 for the guvnor-rest changes, which look logical. 

However, I don't know the UberFire executor code that well, so the real test will be the integration tests.

File: guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java [hunk 1]
```diff
@@ -15,73 +15,72 @@
...
-        this.promoteChangesEvent = promoteChangesEvent;
-        this.releaseProjectEvent = releaseProjectEvent;
-        this.configurationService = configurationService;
+        this.pomService = pomService;
+        this.repositoryService = repositoryService;
         this.projectService = projectService;
     }
 
     @PostConstruct
     public void init() {
```
Comment (manstis) [inline guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java]: Do we need an empty ```@PostConstruct``` method?

File: guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java [hunk 2]
```diff
@@ -91,111 +90,87 @@ public void configureRepository( final String repository,
...
     public Set<Project> getProjects( final Repository repository,
                                      final String branch ) {
         return projectService.get().getProjects( repository,
                                                  branch );
     }
 
-    protected String encodePassword( final String password ) {
-        if ( password == null ) {
-            return null;
+    private Set<Project> getProjects( final Repository repository ) {
```
Comment (manstis) [inline guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java]: The old [code](https://github.com/mswiderski/guvnor/blob/44651ca6a7fa8b147212302b4f47e8408b91c7ad/guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java#L190) would get a ```Set``` of "[authorized](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/AbstractProjectService.java#L145)" projects; this new code simply gets a collection of all Projects in the Repository.

**Comment by manstis** [comment]
A couple of comments.. otherwise looks good to me.

Thanks @mrietveld for looking at the REST related changes too,

File: guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java (hunk 2 above)
Comment (mswiderski) [inline guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java]: that's actually as it should be as this is system operation so all projects should be updated to be consistent

File: guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java (hunk 1 above)
Comment (mswiderski) [inline guvnor-asset-mgmt/guvnor-asset-mgmt-backend/src/main/java/org/guvnor/asset/management/backend/service/AssetManagementServiceImpl.java]: removed

**Comment by manstis** [comment]
Thanks @mswiderski +1 to merge (please let me know when this can be merged).

**Pull request merged**